src/main/scala/lockbook/dev/SettingsHelper.scala

```diff
@@ -0,0 +1,96 @@
+package lockbook.dev
+
+import java.io.File
+import java.util.concurrent.TimeUnit
+
+import io.circe.parser._
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder, HCursor, Json}
+
+import scala.concurrent.duration.FiniteDuration
+
+trait SettingsHelper {
+  def getTheme: Theme
+  def getAutoLock: AutoLock
+}
+
+object SettingsHelper {
+  val jsonPath = s"${App.path}/lockbook.json"
+
+  def fromFile(fileHelper: FileHelper): LockbookSettings =
+    fileHelper
+      .readFile(jsonPath)
+      .flatMap(decodeSettings) match {
+      case Left(tempError) =>
+        if(new File(jsonPath).exists()) println(s"Could not read settings: $tempError")
```

Comment:
Instead of doing this check you can just `println(tmpError.uiMessage)`

also `tempError` -> `error`